### ISSUE TITLE
Holding backspace on chrome OS doesn't send repeated key presses

### ISSUE THREAD

**Issue by mkokotovich** [termux/termux-app#156]
In the rest of Chrome OS (and every other OS) when I press and hold backspace it eventually starts to send repeated keystrokes. In termux it doesn't seem to send the repeats, just one per press. I'd love to be able to backspace quickly by holding down the backspace key.

**Comment by qntmpkts** [comment]
Title should be modified to "Holding any key...".

This may be an issue with how Termux and other Android apps handle key inputs for public class KeyEvent: https://developer.android.com/reference/android/view/KeyEvent.html .
For example, on some Android apps in Chrome OS, the key-repeat-on-hold works as expected but on others it doesn't. I haven't done a code review of the repo in lieu of the maintainer, so just mentioning for now that this might be the underlying cause.

**Comment by qntmpkts** [comment]
According to rumors (because I cannot find hard sourcing), this is a known issue in the Android Runtime for Chrome and is being actively debugged. I will update this post if I happen to find any info.

In the meantime, it is advisable to submit feedback to the devs with Alt+Shift+i.

FYI, I report that *any key* fails on samus (pixel). This may not reflect everyone's experience, however.

**Comment by mkokotovich** [comment]
Agreed, that is my experience as well. Any key press is failed to be repeated. I have an asus flip.

**Comment by justinkelly** [comment]
Just retested this - on my ChromeBook Flip - Beta channel - holding any key now repeats!!

1 in 10 times it wont work perfectly - but its now good enough that I'm happily coding away in Termux on ChromeOS :)

**Comment by sneak** [comment]
Using Chrome OS `Version 63.0.3239.140 (Official Build) (64-bit)` I get key repeats, but the rate is slow - it doesn't obey the system key repeat rate setting in Chrome OS.
